src/workers/postUpdated.ts

```diff
@@ -107,64 +108,73 @@ const createPost = async ({
   mergedKeywords,
   questions,
 }: CreatePostProps) => {
-  const existingPost = await entityManager
-    .getRepository(Post)
-    .createQueryBuilder()
-    .select('id')
-    .where(
-      'url = :url or url = :canonicalUrl or "canonicalUrl" = :url or "canonicalUrl" = :canonicalUrl',
-      { url: data.url, canonicalUrl: data.canonicalUrl },
-    )
-    .getRawOne();
-  if (existingPost) {
-    logger.info({ data }, 'failed creating post because it exists already');
-    return null;
-  }
+  const meter = opentelemetry.metrics.getMeter('api-bg');
+  return runInSpan('createPost', async () => {
```

Comment:
We don't necessarily need this span. Tracing is not required but I don't really mind keeping it, your call

Reply:
Just this one, or in general? 
As it's only fraction of calls that get traced wasn't sure if the spans adding anything over the counters alone?

Reply:
It's usually good for performance benchmarking and so on. If we care about optimizing the speed of this call or willing to know if anything goes south in networking

Reply:
I see then the counter alone should do it. We only care about the 2 edge cases returns and how often they happen.